### DISCUSSION TITLE
Collab plugin is not triggering updates

### DISCUSSION THREAD

**Discussion by allan-maeots** [Milkdown/milkdown#1182] [Q&A]
Is there a reason why Collab plugin does not trigger listeners (updated / markdownUpdated) when another user alters the document? 

This works when using YJS with CodeMirror for example (`EditorView.updateListener.of((viewUpdate) => { ... `) so the listener is triggered every time no matter who types something.

**Comment by lexhuismans-wk** [comment]
We need this for saving content, without two users will see the same content but are storing different (outdated) ones... Do you know if there is a work-around?
